README.md

```diff
@@ -0,0 +1,4 @@
+# README
+
+## Running the code
+"just do `cargo run --release`"
```

Comment:
It wouldn't hurt to outline the reasons for the --release flag

Reply:
this is a direct copy/paste from your message to me so if anyone is to blame here, it's you.